cmd/satellite/open_stream.go

```diff
@@ -61,17 +62,26 @@ func NewOpenStreamCommand() *cobra.Command {
 			if err := flags.ValidateAll(); err != nil {
 				return err
 			}
-			_, err := time.Parse("2006-01-02'T'15:04:05Z", openStreamFlag.AutoCloseTime)
-			if err != nil {
-				return err
+			if openStreamFlag.EnableAutoClose {
+				if openStreamFlag.AutoCloseTime == "" {
+					return errors.New("cannot enable auto close without providing an auto close time")
+				}
+				_, err := time.Parse("2006-01-02 15:04:05", openStreamFlag.AutoCloseTime)
+				if err != nil {
+					return errors.New("couldn't parse auto close time. Please use layout 2006-01-02 15:04:05")
+				}
+				if openStreamFlag.AutoCloseDelay < 1*time.Second || openStreamFlag.AutoCloseDelay > 10*time.Minute {
+					return errors.New("please provide a duration between 1s and 10m")
+				}
 			}
+
 			return nil
 		},
 		Run: func(cmd *cobra.Command, args []string) {
 			proxy := proxyFlags.ToProxy()
 			defer proxy.Close()
 
-			autoCloseTime, _ := time.Parse("2006-01-02'T'15:04:05Z", openStreamFlag.AutoCloseTime)
+			autoCloseTime, _ := time.Parse("2006-01-02 15:04:05", openStreamFlag.AutoCloseTime)
```

Comment:
Was the reason for removing the T and Z because other CLI commands also don't use them? (I think using T and Z should parse). Z is very important because it denotes UTC.

Reply:
Yes, I copied the format that's used elsewhere in the CLI.

Reply:
Nope! I just checked other usages (notably in list-plans) and see that it matches this format so we should be consistent. We could change the format in the future but would be best to do it in a blanket change.

I think to use this format we must specify (in UTC) similar to the list-plans command params.